.circleci/config.yml

```diff
@@ -54,16 +54,10 @@ jobs:
             black --check spectacles tests
 
       - run:
-          name: pytest
+          name: pytest and coverage
           command: |
             . venv/bin/activate
-            python -m pytest -p no:randomly --record-mode=none
-
-      - run:
-          name: coverage
-          command: |
-            . venv/bin/activate
-            coverage run --source spectacles -m pytest
+            coverage run --source spectacles -m pytest -p no:randomly --record-mode=none
```

Comment:
I think it's to our benefit to run randomly because it catches issues where one test depends on the state of another. It reveals implicit dependencies between tests due to the ordering.

If we get a flaky test, randomly prints the seed used so it's straightforward to reproduce.
If we experien

Reply:
@joshtemple 👍 Just updated.